### PR TITLE
Import @ovh-ux/ng-ovh-actions-menu

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-4500
| License          | BSD 3-Clause

## Description

Import **@ovh-ux/ng-ovh-actions-menu** :truck:

Using following command:

```sh
lerna import ~/Code/github/ovh-ux/ng-ovh-actions-menu \
  --dest=packages/components \
  --preserve-commit \
  --flatten
```

### :construction_worker_man: Build

ea4a70a - build: remove extra parts

### :memo: Documentation

5bee669 - docs: update README.md file

### :bookmark: TODOs

- [x] Convert XML files to JSON.
- [x] Update CONTRIBUTORS file.

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10424.0 ✘  
* 
  * Build ✘

**Comment by cbourgois** [comment]
Just `yarn.lock` in conflict but it will be the same every time

File: packages/components/ng-ovh-actions-menu/src/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    'ngOvhResponsivePopover',
+    'pascalprecht.translate',
+  ])
+  .directive('actionsMenu', directive)
+  .factory('ActionsMenu', factory)
+  .provider('actionsMenu', provider)
+  .run(/* @ngInject */($templateCache) => {
+    $templateCache.put('ng-ovh-actions-menu/template-inner.html', innerTemplate);
+  })
+  .run(/* @ngTranslationsInject:json ./translations */);
```
Comment (cbourgois) [inline packages/components/ng-ovh-actions-menu/src/index.js]: 👍

**Comment by antleblanc** [comment]
Following Pull Request has suffered several conflicts that's why I close it in
order to re-open a new one up-to-date with `develop` branch.

Thanks!

**Pull request closed without merging**